akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKitScalaTestSpec.scala

```diff
@@ -20,13 +18,9 @@ import org.scalatest.time.Span
  *
  * Note that ScalaTest is not provided as a transitive dependency of the testkit module but must be added explicitly
  * to your project to use this.
- *
- * This is a `WordSpec`, other ScalaTest styles can be implemented in a similar abstract class like this one.
- * For example `with FlatSpecLike with BeforeAndAfterAll` and implementing the `afterAll` method by calling
- * `testKit.shutdownTestKit()`.
  */
-abstract class ActorTestKitWordSpec(testKit: ActorTestKit) extends ActorTestKitBase(testKit)
-  with WordSpecLike with Matchers with BeforeAndAfterAll with ScalaFutures with Eventually {
+abstract class ActorTestKitScalaTestSpec(testKit: ActorTestKit) extends ActorTestKitBase(testKit)
+  with TestSuite with Matchers with BeforeAndAfterAll with ScalaFutures with Eventually {
```

Comment:
mixing in TestSuite fixes the compile looking for run method

Reply:
ah, I thought I had tried all things. Probably got lost in the trait jungle. Thanks